docs/README.md

```diff
@@ -8,7 +8,7 @@ The code and documentation are hosted in the following repository: https://githu
 ```
 p4c
 ├── build                     -- recommended place to build binary
-├── p4include                 -- standard P4 files needed by the compiler
+├── p4include                 -- standard P4 files needed by the compiler (e.g., core.p4)
 ├── backends
```

Comment:
Keep this in alphabetical order?  Makes it easier to avoid accidental duplication.